feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -282,23 +282,19 @@ func testIPv4BackUpSwitch(ctx context.Context, t *testing.T, args *testArgs) {
 	BaseFlow := createFlow(t, args.ate, args.top, "BaseFlow")
 
 	// validate programming using AFT
-	aftCheck(t, args.dut, dstPfx, []string{"192.0.2.6", "192.0.2.10"})
```

Comment:
Replace this with a `TODO`? It seems like we should say that when we have concluded exactly what to mark here then we should come back and add the check.

Reply:
Done.